common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessageView.java

```diff
@@ -7,31 +7,37 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.rest.response;
+package com.ca.mfaas.message.api;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Data;
 
 import java.util.List;
 
 /**
- * This interface is intended for REST API responses that contain error,
+ * This class is intended for REST API responses that contain error,
  * warning, or informational messages in the common MFaaS format.
  *
  * It is preferred to return successful responses without messages if possible
  * and use only plain responses without wrapping for them.
```

Comment:
What means 'plain responses without wrapping for them'? 
It's a bit hard to understand the point of this sentence.

Reply:
Sounds good!